### PR TITLE
Add Track to DB when added to Spotify

### PR DESCRIPTION
**Pull request by VOjha**
Yesterday we added the thing where if you search for a track and hit +, the song gets added to your Spotify playlist. We forgot to make a Track object and add it to the DB... so this PR does that.

I also did a lot of cleanup in this branch, so the only important difference is in `tracks.py`.

Notable change: instead of just getting `track_uri` from the POST request, we now get all the track info (because we need it to make the Track object). Then I used regular expressions (because we get a string of track info) to get out all the relevant things.

@vsiri @kimberlytze

### CODE REVIEW

File: upnext/tracks.py [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
-  sp = spotipy.Spotify(auth=token_info['ACCESS_TOKEN'])
-  results = sp.user_playlist_add_tracks(username, party_id, [track_id])
-  print results
+def add_to_playlist(track_info, party):
+    token_info = tokens.token_read()
+    # Adding on Spotify
+    track_uri = re.search('uri\': u\'(.+?)\',', track_info).group(1)
+    username = 'up--next'
+    track_id = track_uri.split(':')[-1]
+    print track_id, "track id"
```
Comment (vsiri) [inline upnext/tracks.py]: do we want these print statements still

File: upnext/tracks.py [hunk 2]
```diff
@@ -1,34 +1,39 @@
...
+    party_id = party.uri.split(':')[-1]
+    sp = spotipy.Spotify(auth=token_info['ACCESS_TOKEN'])
+    results = sp.user_playlist_add_tracks(username, party_id, [track_id])
+    print results, "result of add track"
```
Comment (vsiri) [inline upnext/tracks.py]: ^

File: upnext/refresh.py [hunk 3]
```diff
@@ -1,31 +1,27 @@
...
 from threading import Thread
-import spotipy.util as util
 import spotipy.oauth2 as oauth2
 import tokens
 
 
 class Refresh(Thread):
-  def refresh_tokens(self):
-    now = time.strftime("%c")
-    print "I refreshed at", now
```
Comment (vsiri) [inline upnext/refresh.py:14]: can we keep the now = time... and print statement for debugging purposes on localhost + making sure it runs properly

**Comment by vsiri** [comment]
comments then 🐑 !! ayyy

File: upnext/refresh.py (hunk 3 above)
Comment (VOjha) [inline upnext/refresh.py:14]: I gotchu

**Comment by kimberlytze** [comment]
LGTM :rocket:

**Pull request closed without merging**